fs/fs.go

```diff
@@ -117,14 +117,27 @@ func NewFsInfo(context Context) (FsInfo, error) {
 
 func (self *RealFsInfo) addLabels(context Context) {
 	dockerPaths := getDockerImagePaths(context)
+
+	// Determine if Docker is using devicemapper for its storage driver. If so, don't invoke
+	// self.updateDockerImagesPath below, as that will incorrectly override the device associated with
+	// LabelDockerImages (set previously up above in NewFsInfo).
+	dockerUsesDeviceMapper := false
+	for dev, p := range self.partitions {
+		if p.fsType == "devicemapper" && self.labels[LabelDockerImages] == dev {
```

Comment:
Wouldn't this be better done using the `StorageDriver` value from `docker info`?